gtp2ogs.js

```diff
@@ -418,34 +418,84 @@ process.on('uncaughtException', function (er) {
   }
 })
 
+// console messages : for bot admins information
+
+console.log("You are using gtp2ogs version 6.0 \n - For changelog or devel updates, please visit : https://github.com/online-go/gtp2ogs/tree/devel \n - For known issues, please visit https://github.com/online-go/gtp2ogs/KNOWN-ISSUES.md");
+
+// console warnings : for bot admins usage
+
+if (argv.maxhandicap && (argv.maxhandicapranked || argv.maxhandicapunranked)) {
+    console.log("Warning: You are using --maxhandicap in combination with --maxhandicapranked or/and --maxhandicapunranked. \n Use either --maxhandicap alone , OR --maxhandicapranked with --maxhandicapunranked. \n But don't use the 3 --maxhandicap arguments at the same time. " );
```

Comment:
Please change this to "and/or", as that's correct english.

Reply:
Also remove the extra space before `\n`, that extra space risks console to turn over at the space, resulting in a double newline.

Reply:
done :1st_place_medal: